_posts/2022-06-17-ALP-Community-update.md

```diff
@@ -0,0 +1,36 @@
+---
+
+author: Lubos Kocman
+date: 2022-06-17 12:00:00+00:00
+layout: post
+license: CC-BY-SA-3.0
+title: Community work group update post oSCON22
+image: /wp-content/uploads/2022/06/opensuse-alp-concept-unofficial.png
+categories:
+- Announcements
+- openSUSE
+- Community
+tags:
+- openSUSE
+- Open Source
+- Community
+- Work Group
+- Open
+- Linux
+- AdaptableLinuxPlatform
+
+---
+
+[Community workgroup](https://en.opensuse.org/openSUSE:ALP/Workgroups/Community) for ALP would like to update you on what has happened since the openSUSE Conference 2022. Make sure to check the panel discussion with the ALP steering committee and other [talks](https://www.youtube.com/playlist?list=PL_AMhvchzBae22JG6CeBsSpjdrgMaeyN7) from oSCON22 as they become public.
+
+First, there is a new official communication channel! We're using the "openSUSE ALP" matrix channel, bridged with https://discord.gg/opensuse and ircs://libera.chat/#opensuse-alp. Whatever you use, we've got you covered! This is the best way how to consume updates.
+
+We have revisited the way how we'd like to [communicate](https://news.opensuse.org/2022/06/15/community-aims-to-grow-comms-marketing-team/) updates on ALP. The idea is to switch away from the digest report for all workgroups to something a bit easier to follow and more exciting to read.
```

Comment:
Second, we have revisited the way how we'd like to [communicate](https://news.opensuse.org/2022/06/15/community-aims-to-grow-comms-marketing-team/) updates on ALP. The idea is to switch away from the digest report for all workgroups to something a bit easier to follow and more exciting to read.